src/main/java/Duke.java

```diff
@@ -6,5 +8,99 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("\t____________________________________________________________");
+        taskTracker();
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void taskTracker() {
+        Task[] tasks = new Task[100];
+        int indexOfTask = 0;
+        String userInput;
+        Scanner sc = new Scanner(System.in);
+
+        running:
+        while (sc.hasNextLine()) {
+            userInput = sc.nextLine();
+            //Split user input to retrieve task type (i.e. event, deadline) and task description.
```

Comment:
Good documentation and appropriate comments to help developers review your code and hop right into it!